templates/systemd/sentinel.service.erb

```diff
@@ -16,6 +16,8 @@ ExecStart=/usr/bin/redis-sentinel  <%= @sentinel_run_dir %>/<%= @conf_file_name
 ExecStop=/usr/bin/redis-cli -p <%= @sentinel_port %> shutdown
 User=<%= @sentinel_user or 'root' %>
 Group=<%= @sentinel_group or 'root' %>
+Restart=always
+RestartSec=1m
```

Comment:
Let's set it to try 5 times and then give up but wait 10s not 1m, which is too much.
```
[Unit]
StartLimitIntervalSec=100
StartLimitBurst=5

[Service]
RestartSec=10
```

Reply:
Also to consider if should change "always" to "on-failure"